spec/controllers/search_controller_spec.rb

```diff
@@ -195,4 +195,44 @@ def make(let); end
       end
     end
   end
+
+  describe "POST add_material_to_collections" do
+
+    let(:clazz)         { FactoryGirl.create(:portal_clazz,:course => @mock_course,:teachers => [teacher]) }
+    let(:another_clazz) { FactoryGirl.create(:portal_clazz,:course => @mock_course,:teachers => [teacher]) }
+
+    let(:already_assigned_offering) { FactoryGirl.create(:portal_offering, :clazz_id=> clazz.id, :runnable_id=> chemistry_investigation.id, :runnable_type => 'Investigation'.classify) }
+    let(:another_assigned_offering) { FactoryGirl.create(:portal_offering, :clazz_id=> clazz.id, :runnable_id=> laws_of_motion_activity.id, :runnable_type => 'Investigation'.classify) }
```

Comment:
Instead of a Investigation runnable_type you should use ExternalActivity (if possible). We no long support Investigation or Activity runnable_types, only ExternalActivities are supported.